src/locales/en/pokemon-form.ts

```diff
@@ -0,0 +1,190 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const pokemonForm: SimpleTranslationEntries = {
+  // Battle forms
+  "mega": "Mega {{pokemonName}}",
+  "mega-x": "Mega {{pokemonName}} X",
+  "mega-y": "Mega {{pokemonName}} Y",
+  "primal": "Primal {{pokemonName}}",
+  "gigantamax": "G-Max {{pokemonName}}",
+  "eternamax": "E-Max {{pokemonName}}",
+
+  // Starters forms
+  // 1G
+  "pikachuCosplay": "Cosplay",
+  "pikachuCoolCosplay": "Cool Cosplay",
+  "pikachuBeautyCosplay": "Beauty Cosplay",
+  "pikachuCuteCosplay": "Cute Cosplay",
+  "pikachuSmartCosplay": "Smart Cosplay",
+  "pikachuToughCosplay": "Thoug Cosplay",
```

Comment:
```suggestion
  "pikachuToughCosplay": "Tough Cosplay",
```
The same typo is in the `zh_TW` placeholder text too.